packages/manager/apps/pci-object-storage/src/pages/objects/container/object/add-replication/ManageReplicationPage.form.tsx

```diff
@@ -38,7 +38,7 @@ import {
 } from './ReplicationRulePriority.component';
 import { ReplicationRuleStorageClass } from './ReplicationRuleStorageClass.component';
 import { ReplicationRuleApplication } from './ReplicationRuleApplication.component';
-import { TContainer } from '../show/Show.page';
+import { TContainer } from '@/pages/dashboard/BucketPropertiesCard';
```

Comment:
We need to rework dependencies....
Why this component depends on dashboard page?